### PR TITLE
Fix filesystem references under Windows.

### PR DESCRIPTION
**Pull request by pchote**
Fixes scripted maps, and the TS mod crashes under Windows.

### CODE REVIEW

**Comment by reaperrr** [comment]
Fixes shellmaps and I didn't experience any crash in TS, either (Win7x64).

:+1:

File: OpenRA.Game/Map/Map.cs [hunk 1]
```diff
@@ -1178,7 +1178,8 @@ public IEnumerable<CPos> FindTilesInCircle(CPos center, int maxRange, bool allow
 		// Placeholders for future implementation
 		public Stream Open(string filename)
 		{
-			if (Container.Contains(filename))
+			// Explicit paths never refer to a map
+			if (filename.IndexOf('|') == -1 && Container.Contains(filename))
```
Comment (Mailaender) [inline OpenRA.Game/Map/Map.cs]: `filename.IndexOf('|') == -1` translates to `!filename.Contains('I')`?

Comment (pchote) [inline OpenRA.Game/Map/Map.cs]: Yeah, I guess that would be clearer.

**Comment by pchote** [comment]
Updated.

**Pull request merged**